### ISSUE TITLE
Not working on blazor net core preview 9

### ISSUE THREAD

**Issue by arivera12** [nheath99/NodaTimePicker#66]
Is this working/migrated to blazor latest version?
![image](https://user-images.githubusercontent.com/5600102/64497490-8d4e3000-d27c-11e9-81db-a5c803349649.png)

**Comment by nheath99** [comment]
The published version is still on Preview 8 at the moment, will work to get it up to preview 9 as soon as possible. Unfortunately the Azure build agents seem to lag behind with availability of the .Net Core Preview SDKs so the latest Blazor version doesn't always build on Azure immediately.

**Comment by arivera12** [comment]
I see. On this last preview (9) there was a lot of breaking changes. A lot of class and methods were renamed. Also to call events from html tag events must start with '@' like this @onclick="mymethod". Components parameters must be explicitly public, and so on. I just took yesterday to migrate everything. It seems this will be the last breaking changes cause MS said they aren't adding more feature to dot net core, they will be only fixing bugs to make a stable release after this month. Another important thing to get your visual studio working with this preview you have to update to latest preview version available and also upgrade net core sdk and blazor templates. It should not take too much time to you to migrate it. Please let me know when this is migrated. Here is the link with the changes [https://devblogs.microsoft.com/aspnet/asp-net-core-and-blazor-updates-in-net-core-3-0-preview-9/](https://devblogs.microsoft.com/aspnet/asp-net-core-and-blazor-updates-in-net-core-3-0-preview-9/)

**Comment by nheath99** [comment]
Fixed and new NuGet version 0.4.0 available

**Comment by julienGrd** [comment]
Personnaly I don't have the arivera12's problem with this version, the component work fine in 0.4.0, at least for the DatePicker, I don't try the TimePicker.
My only problem is i'm unable to reference the static file in my _Host, i have 404 of these, so for now i downloaded the file and reference them manually
````
<link href="_content/NodaTimePicker/datepicker.css" rel="stylesheet" /> => 404
<link href="_content/NodaTimePicker/timepicker.css" rel="stylesheet" /> => 404
````

**Comment by arivera12** [comment]
This is my visual studio version
![image](https://user-images.githubusercontent.com/5600102/64879518-2508a100-d624-11e9-8cf0-52dcb99aa653.png)

@nheath99 are you using other version? This seems to be the latest version available.

**Comment by nheath99** [comment]
@arivera12 I'm using the same version as you. I have created a new Blazor application and can reproduce your code with one change: Selected has been renamed OnSelected (please see screenshot below). Unfortunately there will be breaking changes until version 1.0.

![image](https://user-images.githubusercontent.com/10851693/64923406-9afe3b00-d7d1-11e9-838a-d328f286b651.png)

If you are still having problems then I suggest you create a new Blazor application (preview 9), copy my code above and compare the differences with your code.

**Comment by nheath99** [comment]
> Personnaly I don't have the arivera12's problem with this version, the component work fine in 0.4.0, at least for the DatePicker, I don't try the TimePicker.
> My only problem is i'm unable to reference the static file in my _Host, i have 404 of these, so for now i downloaded the file and reference them manually
> 
> ```
> <link href="_content/NodaTimePicker/datepicker.css" rel="stylesheet" /> => 404
> <link href="_content/NodaTimePicker/timepicker.css" rel="stylesheet" /> => 404
> ```

@julienGrd I have just resolved this in #69

**Comment by arivera12** [comment]
@nheath99 I have a razor component lib project where I am wrapping some components there. I tried installing it on a blazor project and still the same result. Give me some time to do some research on the web to see if I get this working. I will close this issue in the meantime.
